x/cronos/types/keys.go

```diff
@@ -31,6 +31,9 @@ var (
 	KeyPrefixDenomToExternalContract = []byte{prefixDenomToExternalContract}
 	KeyPrefixDenomToAutoContract     = []byte{prefixDenomToAutoContract}
 	KeyPrefixContractToDenom         = []byte{prefixContractToDenom}
+
+	// ParamsKey is the key for params.
+	ParamsKey = []byte{0x00}
```

Comment:
Should be added the enum with the other key prefixes.

Reply:
done